jdat_notebooks/optimal_extraction/requirements.txt

```
@@ -0,0 +1,7 @@
+numpy
+astropy
+matplotlib
+scipy
+ipywidgets
+jwst
```

As @gkanarek and I discussed, `jwst` pulls v0.0.0 from PyPI. @gkanarek is using jwst version v0.16.1. This is required, as v0.16.2 appears to break this on my end. 
```suggestion
git+git://github.com/spacetelescope/jwst@0.16.1#egg=jwst
```